test/built-ins/Function/prototype/caller-arguments/cross-realm-behavior.js

```diff
@@ -0,0 +1,13 @@
+/*---
+description: Function.prototype caller and arguments properties use the same ThrowTypeError across realms
+features: [cross-realm]
+---*/
+
+const otherRealm = $262.createRealm();
+const otherFunctionProto = otherRealm.evaluate('Function.prototype');
+
+const mainCallerDesc = Object.getOwnPropertyDescriptor(Function.prototype, "caller");
+const otherCallerDesc = Object.getOwnPropertyDescriptor(otherFunctionProto, "caller");
+
+assert.sameValue(mainCallerDesc.get, otherCallerDesc.get, "caller getter should be same across realms");
+assert.sameValue(mainCallerDesc.set, otherCallerDesc.set, "caller setter should be same across realms");
```

Comment:
This is incorrect: `%ThrowTypeError%` is defined once per realm, cf. <https://tc39.es/ecma262/#sec-%throwtypeerror%>.

See also <https://github.com/tc39/test262/blob/main/test/built-ins/ThrowTypeError/unique-per-realm-function-proto.js>.